src/shared/data/Tile-list.es6

```diff
@@ -0,0 +1,183 @@
+'use strict'
+
+//
+// Tile-lis
+// Created by dpekar on 10/7/15.
+//
+
+module.exports = {
+
+    'basic': [
```

Comment:
We might want to make each of these an object instead of an array so that we can grab specific tiles by the id or something similar.  There may be cases where we need to check for specific tile types?  Not sure, just a thought